Lexplorer/Pages/NFTDetail.razor

```diff
@@ -84,6 +84,14 @@
                 <NFTContent nftMetadata="@nftMetadata" />
             </td>
         </tr>
+        <tr>
+            <td colspan="2">
+                @if(nft?.token != null)
+                {
+                    <MudButton Variant="Variant.Filled" Color="Color.Primary" OnClick="goToCollectionPage">View collection</MudButton>
+                }
```

Comment:
Why is only the MudButton covered by the if and not the complete tr?

Reply:
fixed in [a7c8765](https://github.com/fudgebucket27/Lexplorer/pull/182/commits/a7c8765666c9e986a907b574851f6695b1d15a0b)

Reply:
oh golly i need to push another commit to fix the {

Reply:
fixed in [ef5e1e5](https://github.com/fudgebucket27/Lexplorer/pull/182/commits/ef5e1e545f2b6d8e0c869aff02c328612c0b75ed)